runtime/commonTest/src/kotlinx/serialization/json/JsonConfigurationTest.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.json
+
+import kotlin.test.*
+
+class JsonConfigurationTest {
+
+    @Test
+    fun testPrettyPrint() {
+        json(true, "")
```

Comment:
Can you add tests on the correctness of floating point-related exceptions?

Reply:
We already have `SpecialFloatingPointValuesTest`